.eslintrc

```diff
@@ -22,5 +22,6 @@
       "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
       "react/forbid-prop-types": "off",
       "react/no-array-index-key": "off",
+      "linebreak-style": "off",
```

Comment:
Oh yes, thanks for the reminder. I did try it once, briefly, but I think it didn't work before because I didn't commit it? Or maybe there is something else I need to do to make git really refresh the line endings on my end. I guess worst case, I commit and then delete my local copy of the repo.